R/weight.R

```diff
@@ -188,3 +341,15 @@ generateWeightEntry <- function (expr) {
         targets = target_list
     ))
 }
+
+#' Validate that a value is TRUE or FALSE
+#'
+#' @param value The right hand side of an assignment
+#' @keywords internal
+#' @return nothing, called for its side effects
+#'
+validateWeightVariableValue <-  function (value) {
```

Comment:
How about just "is.TRUEorFALSE" or something? (And then don't say "Right hand side" in the error message). Then we can use this elsewhere where we require a single T/F value.